projects/1.21/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -0,0 +1,2009 @@
+  "ars_nouveau.perk.max_mana": "最大魔力",
+  "ars_nouveau.perk.max_mana.desc": "增加魔力的最大值。",
+  "ars_nouveau.perk.max_perks": "此盔甲的能力数量已达最大值。",
+  "ars_nouveau.perk.not_perk": "对改衣台使用纤维以将其编入盔甲。",
+  "ars_nouveau.perk.percent_max_mana": "最大魔力",
+  "ars_nouveau.perk.saturation": "额外饱和度",
+  "ars_nouveau.perk.saturation.desc": "增加食物带来的饱和度。",
+  "ars_nouveau.perk.set_armor": "请先在台上放置盔甲。",
+  "ars_nouveau.perk.spell_damage": "法术强度",
+  "ars_nouveau.perk.spell_damage.desc": "增加新生魔艺法术的伤害。",
+  "ars_nouveau.perk.warding": "守护",
+  "ars_nouveau.perk.warding.desc": "减少所受的魔法伤害。",
+  "ars_nouveau.perk.wixie": "药水持续时间",
+  "ars_nouveau.perk.wixie.desc": "延长药水效果的持续时间。",
+  "ars_nouveau.perk_desc.thread_amethyst_golem": "每级给予15%%的击退抗性。",
+  "ars_nouveau.perk_desc.thread_chilling": "伤害类效果构件在产生法术效果前会给予目标冰冻效果。每级增加10秒的冰冻效果。3级槽位中变为冰冻 II。",
+  "ars_nouveau.perk_desc.thread_depths": "减少消耗氧气的频率，以此大幅增加安全潜水的时间。能力处于3级或以上槽位时，不会再消耗空气，且游泳速度大幅提升。可与水下呼吸魔咒叠加。",
+  "ars_nouveau.perk_desc.thread_drygmy": "额外给予一个堆叠的掉落物。",
+  "ars_nouveau.perk_desc.thread_feather": "按百分比减少摔落伤害。可与摔落缓冲魔咒叠加。",
+  "ars_nouveau.perk_desc.thread_gliding": "可让你如同佩戴鞘翅一般滑翔。必须编入3级或以上槽位。",
+  "ars_nouveau.perk_desc.thread_heights": "可让你跳得更高，并增加安全摔落距离。",
+  "ars_nouveau.perk_desc.thread_high_step": "每级增加1格的最大行走高度。",
+  "ars_nouveau.perk_desc.thread_immolation": "若穿戴者处于着火状态或身处火中，则其施法时会熄灭火焰，并给予焚祭药水效果。火焰类法术会造成额外伤害，持续时间更长；焚祭效果等级越高，增益越大。",
+  "ars_nouveau.perk_desc.thread_kindling": "伤害类效果构件在产生法术效果前会令目标着火一段时间。每级增加5秒的燃烧时间。",
+  "ars_nouveau.perk_desc.thread_life_drain": "通过法术造成伤害会治疗自身，每级增加所造成伤害20%%的治疗量。",
+  "ars_nouveau.perk_desc.thread_magic_capacity": "每级令穿戴者的最大魔力增加10%%。",
+  "ars_nouveau.perk_desc.thread_repairing": "穿戴者可随时间消耗魔源以修复任何种类的魔法盔甲和巫师系列物品。等级越高，修复的速度越快。此能力会作用于所有有关物品，而非仅作用于物品自身。处于3级或以上槽位时，穿戴的所有魔法盔甲不可破坏。",
+  "ars_nouveau.perk_desc.thread_spellpower": "每级给予额外的法术伤害提升。",
+  "ars_nouveau.perk_desc.thread_starbuncle": "每级增加20%%的玩家速度。",
+  "ars_nouveau.perk_desc.thread_undying": "每次睡眠后可抵消一次死亡，如同持有不死图腾一样。需要3级槽位。",
+  "ars_nouveau.perk_desc.thread_warding": "每级以固定量降低所受的魔法伤害。",
+  "ars_nouveau.perk_desc.thread_whirlisprig": "每级增加30%%的食物饱和度。",
+  "ars_nouveau.perk_desc.thread_wixie": "每级延长15%%的药水持续时间。",
+  "ars_nouveau.perks.duplicated": "你已装备的盔甲中有相同的能力。重复能力的效果不会重复触发。",
+  "ars_nouveau.position": "X：%1$d Y：%2$d Z：%3$d",
+  "ars_nouveau.potion_crafting": "药水合成",
+  "ars_nouveau.potion_diffuser.bind_to_jar": "使用散布器前需先绑定至药水罐。",
+  "ars_nouveau.potion_diffuser.no_pos": "使用支配之杖将药水罐绑定至散布器。",
+  "ars_nouveau.potion_diffuser.off": "散布器已关闭。",
+  "ars_nouveau.potion_diffuser.set_pos": "散布器会向附近实体散布药水。",
+  "ars_nouveau.powered_from": "正在接收%s个中转器的信号。",
+  "ars_nouveau.present.from": "%s 赠礼",
+  "ars_nouveau.present.give": "把这个送给朋友，他们会获得额外物品，也可以自己打开！",
+  "ars_nouveau.production": "生产",
+  "ars_nouveau.reagent": "试剂：",
+  "ars_nouveau.recorded_by": "%s 著",
+  "ars_nouveau.recorded_codex": "你在手抄本里记下了你知道的魔符。",
+  "ars_nouveau.relay.current_power": "当前信号：%d",
+  "ars_nouveau.relay.no_from": "抽取位置未设置。",

```

```suggestion
  "ars_nouveau.relay.no_from": "未设置抽取位置。",

```
我记得前文是未设置在前，保持翻译格式的整体统一吧。